cloud/aws/modules.tf

```diff
@@ -13,6 +14,9 @@ module "vast_server" {
   task_cpu    = 2048
   task_memory = 4096
 
+  storage_type        = var.vast_server_storage_type
+  storage_mount_point = "/var/lib/vast"
```

Comment:
Curious to hear @dominiklohmann's or @tobim's opinion on hardcoding this vs. making the DB dir configurable. I personally don't have a strong opinion.

Reply:
IMHO this is more of an implementation detail of the deployment.

Reply:
I agree with @dispanser here, this is an implementation detail of the deployment, and `/var/lib/vast` sounds like a good value for that.

Reply:
Also, the approach here is to avoid having too many different input variables to keep things as simple as possible for a newcomer.

Reply:
All good points. KISS.